src/css/styles/_top.scss

```diff
@@ -0,0 +1,126 @@
+.top {
+  position: relative;
+}
+
+.nav {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: calc(100% - 2 * 2rem);
+  margin: 0 2rem;
+  font-size: 0.6rem;
+  border-bottom: solid $col-orange 3px;
+}
+
+.nav-logo {
+  float: left;
+  margin: 1rem 0;
+}
+
+.nav-menu {
+  float: right;
+}
+
+.nav-menu__list {
+  display: inline-block;
+  margin: 1rem;
+}
+
+.nav-menu__list__item {
+  display: inline;
+}
```

Comment:
BEM: please use `nav-menu__list-item` or `nav__menu-list-item` instead of `nav-menu__list__item` (http://getbem.com/naming/)